src/index.html

```diff
@@ -3,10 +3,239 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <title>Product</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="use-credentials"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="/">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--is-active">
+            <a href="/" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="/" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+
+      <div class="card" data-qa="card">
+        <img src="./images/imac.jpeg" alt="imac" class="card__img">
+        <a href="/imac" class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</a>
+        <div class="card__code">Код товара: 195434</div>
+        <div class="card__response">
+          <div class="card__stars-4">
```

Comment:
```suggestion
          <div class="card__stars card__stars--4">
```
Один основной класс, и класс-модификатор